src/server/index.tsx

```diff
@@ -158,6 +158,16 @@ export const createServer = ({
   // Return 200 for csp reports - this will need to be overridden when deployed.
   app.post('/__cspreport__', (req, res) => res.status(200).end('ok'));
 
+  app.get('/__version__', (req, res) => {
+    const versionFile = path.join(rootPath, 'version.json');
+
+    return res.sendFile(versionFile);
+  });
+
+  app.get(['/__heartbeat__', '/__lbheartbeat__'], (req, res) => {
```

Comment:
The spec [says](https://mana.mozilla.org/wiki/display/SVCOPS/New+Service+Guide#NewServiceGuide-/__heartbeat__) that `__heartbeat__` should check dependent services. Is there an API endpoint we can check? We definitely depend on the API. This could be done in a follow-up patch if that's easier.

Reply:
I made a wrong assumption in my PR description. I cannot find the API endpoint. I naively thought it would be http://addons.mozilla.org/__heartbeat__, but that does not work, heh.

Otherwise I would have suggested the following: we could maybe proxy the request to `https://<api>/__heartbeat__`?

Reply:
So, AMO does not have these endpoints (yet?). I am not sure which API endpoint we could use to check whether the API is healthy, so I'd like to defer than in another issue: https://github.com/mozilla/addons-code-manager/issues/524

Reply:
I think it's OK for now, at least from ops perspective. We'll be using the __lbheartbeat__ to start with.